docs/.vuepress/config/locales/tr/index.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+    lang: 'tr',
+    title: 'Flarum Dokümantasyon',
```

Comment:
"Doküman" Fransızca bir kelimedir. Onun yerine Türkçe olan "Belge" kelimesini kullanmayı tercih edebilirsin. 
[Source](https://sozluk.gov.tr/)

Reply:
Thanks, fixed it.

- https://github.com/flarum/docs/pull/154/commits/bb0c788c46394d8acac0a45c1172ffc627dfd33e